apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.story.jsx

```diff
@@ -52,13 +52,27 @@ export default storybook => {
     .storiesOf('SectionAssessments/MultipleChoiceOverviewTable', module)
     .addStoryTable([
       {
-        name: 'Table for assessments',
+        name: 'Assessment multiple choice with 7 answers',
         description: 'Ability to see assessment overview for the entire class',
         story: () => (
             <MultipleChoiceOverviewTable
               questionAnswerData={multipleChoiceData}
             />
         )
       },
+      {
+        name: 'Assessment multiple choice with 3 answers',
+        description: 'Ability to see assessment overview for the entire class',
+        story: () => (
```

Comment:
super tiny nit: instead of class, can we call this a section to be consistent with other references?

Reply:
Fixed.